app/channels/map_channel.rb

```diff
@@ -0,0 +1,41 @@
+class MapChannel < ApplicationCable::Channel
+  def subscribed
+    map = Map.find(params[:id])
+    stream_for map
+  end
+
+  def move_map(data)
+    map = Map.find(data["map_id"])
+    return if map.x == data["x"] && map.y == data["y"]
+
+    if map.update(x: data["x"], y: data["y"])
```

Comment:
Not suggesting you change this, but some of the examples I've been taking a look at use strong params here in the channels. Something to consier as these get more complex

Reply:
Yeah, I think that makes sense to do if we're updating more than one or two attributes.